content/glossary/end-to-end-encrypted/contents.lr

```diff
@@ -4,9 +4,9 @@ term: end-to-end encrypted
 ---
 definition:
 
-Transmitted data which is [encrypted](../encryption) from origin to destination is called end-to-end encrypted.
+Transmitted data which is [encrypted](../encryption) directly from origin to destination (to exclude third party management) is called end-to-end encryption. End-to-End Encryption (E2EE) relies on an exchange of public keys (used to encrypt). A separate complimentary private key decrypts data/messages for reading by intended party only (end-to-end-encrypted).
```

Comment:
Can you try to explain what you want to say in more simple terms? The glossary is meant as a quick introduction for people that is not very knowledgeable.

Reply:
I see what you are saying. 

I simplified it down to one additional sentence. Adding only what I believe may be a helpful additional piece of info.